runtime/dolphin/src/lib.rs

```diff
@@ -838,6 +876,23 @@ impl_runtime_apis! {
 		}
 	}
 
+    impl nimbus_primitives::NimbusApi<Block> for Runtime {
```

Comment:
I'm a bit at a loss here, I can't seem to implement the runtime APIs correctly.
@Dengjianping or @ghzlatarev could you please have a look what could be wrong with these impls?
Getting a unsatisfied trait bound for the two new impls, but it looks fine to me.
```
error[E0277]: the trait bound `dolphin_runtime::RuntimeApi: nimbus_primitives::AuthorFilterAPI<sp_runtime::generic::Block<sp_runtime::generic::Header<u32, BlakeTwo256>, OpaqueExtrinsic>, nimbus_primitives::nimbus_crypto::Public>` is not satisfied
   --> node/src/command.rs:461:7
    |
461 |                         dolphin_runtime::RuntimeApi,
    |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `nimbus_primitives::AuthorFilterAPI<sp_runtime::generic::Block<sp_runtime::generic::Header<u32, BlakeTwo256>, OpaqueExtrinsic>, nimbus_primitives::nimbus_crypto::Public>` is not implemented for `dolphin_runtime::RuntimeApi`
    |
    = help: the trait `nimbus_primitives::AuthorFilterAPI<__SR_API_BLOCK__, nimbus_primitives::nimbus_crypto::Public>` is implemented for `dolphin_runtime::RuntimeApiImpl<__SR_API_BLOCK__, RuntimeApiImplCall>`
note: required by a bound in `service_nimbus::start_parachain_node`
   --> node/src/service_nimbus.rs:421:11
    |
408 | pub async fn start_parachain_node<RuntimeApi, Executor>(
    |              -------------------- required by a bound in this
...
421 |         + nimbus_primitives::AuthorFilterAPI<Block,NimbusId>
    |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `service_nimbus::start_parachain_node`

error[E0277]: the trait bound `dolphin_runtime::RuntimeApi: nimbus_primitives::NimbusApi<sp_runtime::generic::Block<sp_runtime::generic::Header<u32, BlakeTwo256>, OpaqueExtrinsic>>` is not satisfied
   --> node/src/command.rs:461:7
    |
461 |                         dolphin_runtime::RuntimeApi,
    |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `nimbus_primitives::NimbusApi<sp_runtime::generic::Block<sp_runtime::generic::Header<u32, BlakeTwo256>, OpaqueExtrinsic>>` is not implemented for `dolphin_runtime::RuntimeApi`
    |
    = help: the trait `nimbus_primitives::NimbusApi<__SR_API_BLOCK__>` is implemented for `dolphin_runtime::RuntimeApiImpl<__SR_API_BLOCK__, RuntimeApiImplCall>`
note: required by a bound in `service_nimbus::start_parachain_node`
   --> node/src/service_nimbus.rs:422:11
    |
408 | pub async fn start_parachain_node<RuntimeApi, Executor>(
    |              -------------------- required by a bound in this
...
422 |         + nimbus_primitives::NimbusApi<Block>
    |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `service_nimbus::start_parachain_node`
```

Reply:
**UPDATE: Nevermind, no need to c I seem to be missing a `sp_api::decl_runtime_apis`**

Reply:
https://github.com/Manta-Network/Manta/commit/72b3b2de6e0975a332b36feb98b144f6a215f4e2